manifests/unison/server.pp

```diff
@@ -6,20 +6,17 @@
   String  $root,
   Array[String] $paths,
   Array[String] $filesystems,
+  String $home = '/root',
   Integer $port = 2647
 ) {
-  include nebula::profile::unison::server
+  ensure_packages(['unison'])
```

Comment:
How does this relate to the deleted include? I'm also not sure what `ensure_packages` does and can't find it in the puppet docs. Why this over a `package{}` stanza?

Reply:
`ensure_packages` will create the package resource if it doesn't already exist; unlike a straight package resource it doesn't cause an error if there are multiple `ensure_packages` with the same package: https://forge.puppet.com/puppetlabs/stdlib#ensure_packages

Reply:
I had the same question, but I found `ensure_packages` in the docs and it is apparently preferred when you have multiple resources that depend on a package and don't want a dependency between them (neither is really primary). It's for mitigation of potential duplicate resources, which would tank the catalog.

Reply:
Yes. There are options we could use to avoid `ensure_packages` if we needed to (i.e. put the package resource in the main unison profile) rather than doing this in a defined type, but this doesn't hurt anything.